examples/deepwalk/model.py

```diff
@@ -28,53 +28,85 @@ def __init__(self,
                  neg_num=5,
                  num_emb_part=1,
                  sparse=False,
-                 sparse_embedding=False):
+                 sparse_embedding=False,
+                 shared_embedding=False):
         super(SkipGramModel, self).__init__()
 
         self.num_nodes = num_nodes
         self.neg_num = neg_num
+        self.shared_embedding = shared_embedding
 
         # embed_init = nn.initializer.Uniform(
         # low=-1. / math.sqrt(embed_size), high=1. / math.sqrt(embed_size))
-        embed_init = nn.initializer.Uniform(low=-1.0, high=1.0)
-        emb_attr = paddle.ParamAttr(
-            name="node_embedding", initializer=embed_init)
+        u_embed_init = nn.initializer.Uniform(low=-1.0, high=1.0)
+        u_emb_attr = paddle.ParamAttr(
```

Comment:
To load the correct trained embeddings, the parameter attribute _name_ here should be the same as [that](https://github.com/PaddlePaddle/PGL/blob/5f5c3e907db6c9bad6c22c8fe2b5e7f50df41cea/examples/deepwalk/multi_class.py#L58) in the class _Model_ of multi_class.py. Please unify them.

Reply:
Here? I have already changed this in my first commit, also use the name 'u_node_embedding').
![image](https://user-images.githubusercontent.com/60515999/130199467-1aae2f8c-432e-4994-a3fd-54f15dd5c4b3.png)